script/github-actions/stylelint-annotation-format.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const sassLintReport = path.join(__dirname, '../../stylelint-report.json');
+
+if (!fs.existsSync(sassLintReport)) {
+  console.log('No sass-lint report found. Skipping.');
+  return;
+}
+
+const sassLintData = JSON.parse(fs.readFileSync(sassLintReport));
+
+sassLintData.forEach(fileMessages => {
+  if (!fileMessages.errored) return;
+
+  for (const [key, value] of Object.entries(fileMessages)) {
```

Comment:
There are now a few places where linting gets outputted to: `deprecations`, `invalidOptionWarnings`, `parseErrors`, & `warnings`